src/Form/FormRadioGroupFeedback.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import FormText, { resolveTextType } from './FormText';
+import { FORM_TEXT_TYPES } from './constants';
+import { useRadioGroupContext } from './FormRadioGroupContext';
+
+const FormRadioGroupFeedback = ({ children, ...props }) => {
+  const {
+    isInvalid,
+    isValid,
+    getNewDescriptorId,
+    addNewDescriptorId,
+    removeDescriptorId,
+  } = useRadioGroupContext();
+  const [id, setId] = useState(props.id);
+  useEffect(() => {
```

Comment:
Would it make sense to package this useEffect block into a reusable hook?  Looks like it's done in at least this component and the group legend below.

Reply:
(I don't feel strongly and defer to your opinion in the end)

Reply:
feels like a great hook to make.